### PR TITLE
Update Property Name

### PR DESCRIPTION
**Pull request by dihan0604**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [x] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

### CODE REVIEW

File: arm-network/2016-12-01/swagger/serviceCommunity.json [hunk 1]
```diff
@@ -93,7 +93,11 @@
     },
     "BgpServiceCommunityPropertiesFormat": {
       "properties": {
-        "communities": {
+	    "serviceName": {
+          "type": "string",
```
Comment (amarzavery) [inline arm-network/2016-12-01/swagger/serviceCommunity.json]: indentation - incorrect. Open the file in vscode and type Alt + Shift + F

**Comment by amarzavery** [comment]
Model validation is failing: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/207268223#L2522

**Comment by amarzavery** [comment]
@dihan0604 @deepakrajendranmsft - Won't this cause breaking changes in the generated sdk?

**Comment by dihan0604** [comment]
@amarzavery , ya this will be breaking changes from API but not from customer. The feature have not enabled and published.

File: arm-network/compositeNetworkClient.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "./2016-12-01/swagger/networkWatcher.json",
     "./2016-12-01/swagger/publicIpAddress.json",
     "./2016-12-01/swagger/routeFilter.json",
+    "./2016-12-01/swagger/network.json"
```
Comment (amarzavery) [inline arm-network/compositeNetworkClient.json]: This file is already present on line 11.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**